### DISCUSSION TITLE
I'm loving this

### DISCUSSION THREAD

**Discussion by ClintFrancis** [xandramax/Volume1#24] [General]
Wow, what a great and intuitive way to make FM in Euro.
Any chance you're considering making a physical versions of this module (either yourself or as a collar with a builder???

**Comment by xandramax** [comment]
I'm so glad to hear that you're enjoying it!

I've taken a lot of time in designing the interface of Algomorph Advance to make it seem at least hardware-plausible, all the way down to including a button that really only has a use in the case of hardware... To make it a reality would involve some substantial hurdles, such as the price inherent in a module with 5+ DAC channels as well as probable latency concerns. I would absolutely love to see it though.

To anyone with hardware experience who would like to collaborate on such a thing: let's get in touch!

**Reply by AlekseyPyatin** [reply]
Good time of day... help me install this tool in the rack, I did everything as in the instructions, but the rack does not see it...what should I do?

**Reply by xandramax** [reply]
Hello, sorry to hear that you're having trouble!

To be sure, did you follow the directions [found here](https://vcvrack.com/manual/Installing#Installing-plugins-not-available-on-the-VCV-Library) under the heading "Installing plugins not available on the VCV Library"?

Also, are you certain that you downloaded the appropriate file from [the latest release](https://github.com/Delexander/Volume1/releases/)? The file should have the extension ".vcvplugin" and should match your operating system ("win" for Windows, "mac" for macOS, or "lin" for Linux).

**Reply by dcarltonseven** [reply]
I too am having issues getting it working here on VCV Free 2.2.2 (win) 
It does seem to unpack, but shows in list greyed out and is not selectable. It does not show when you search modules. No notable errors appear in the log. (Win10 22H2, 64 gig ram) [and yes, this module looks amazing]

**Reply by dcarltonseven** [reply]
Final fix and happy outcome:
I believe I had an old .json file from an earlier beta (either entries in settings.json or DelexanderVol1.json)  Once I edited out the old entries and removed the other, it came up correctly. Thanks again!
